terraform/modules/tf_stream_alert/main.tf

```diff
@@ -13,6 +13,11 @@ resource "aws_lambda_function" "streamalert_rule_processor" {
   source_code_hash = "${base64sha256(file(var.filename))}"
   publish          = true
 
+  vpc_config {
+    subnet_ids      = ["${var.vpc_subnets}"]
```

Comment:
since these vars are of list type, you should be able to omit the square brackets wrapping these (I could also be misinterpreting this)